docs/sources/reference/commandline/cli.md

```diff
@@ -452,6 +452,27 @@ https://linuxcontainers.org/) via the `lxc` execution driver, however, this is
 not where the primary development of new functionality is taking place.
 Add `-e lxc` to the daemon flags to use the `lxc` execution driver.
 
+#### Exec driver options
+
```

Comment:
I've seen `exec driver`, `exec-driver`, and `execdriver` so I arbitrarily picked `execdriver` we should be consistent unless one is a proper name or a command.  I've seen execdriver in other docker docs  and the .go file uses execdriver it seems.  Anyway, I'd pick one (and I'll update my last comment). Finally, I changed the heading to be more specific --- no sense in getting folks hopes up.

Don't know if that capper sentence about the daemon is necessary. Your call.

-->

#### Options for the native execdriver

You can configure the `native` (libcontainer) execdriver using options specified
with the `--exec-opt` flag. All the flag's options have the `native` prefix. A
single `native.cgroupdriver` option is available.

The `native.cgroupdriver` option specifies the management of the container's cgroups. You can specify
`cgroupfs` or `systemd`. If you specify `systemd` and it is not available, the
system uses `cgroupfs`. By default, if no option is specified, the execdriver
first tries `systemd` and falls back to `cgroupfs`. This example sets the
execdriver to `cgroupfs`:

  	 $ sudo docker -d --exec-opt native.cgroupdriver=cgroupfs
  	 
Setting this option applies to all containers the daemon launches.